### PR TITLE
vb: support complex parameters with typename

### PR DESCRIPTION
**Pull request by ThomasZitzler**

### CODE REVIEW

File: src/XamlTestCases/HelloWorldConsole2.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Activities;
+
+namespace XamlTestCases
+{
+    public class HelloWorld2Input
```
Comment (lbargaoanu) [inline src/XamlTestCases/HelloWorldConsole2.cs]: I'm sure we can find better names here :)

File: src/XamlTestCases/XamlTests.cs [hunk 2]
```diff
@@ -174,5 +174,29 @@ public void XamlWorkflowWithInputsOutputs()
...
+
+        [Fact]
+        public void XamlWorkflowWithInputObject()
+        {
+            var xamlString = @"
+                    <Activity x:Class=""WFTemplate""
+                              xmlns=""http://schemas.microsoft.com/netfx/2009/xaml/activities""
+                              xmlns:s=""clr-namespace:System;assembly=mscorlib""
+                              xmlns:s1=""clr-namespace:System;assembly=System""
+                              xmlns:sa=""clr-namespace:CoreWf;assembly=CoreWf""
```
Comment (lbargaoanu) [inline src/XamlTestCases/XamlTests.cs]: Does it work without this line? Most things were renamed to System.Activities.

**Comment by lbargaoanu** [comment]
Looks good, but a few minor things.

File: src/XamlTestCases/XamlTests.cs (hunk 2 above)
Comment (ThomasZitzler) [reply src/XamlTestCases/XamlTests.cs]: You are right, I started with it before you renamed and did copy + paste of the old one to get started. Works with "System.Activities"

**Review by lbargaoanu — APPROVED** [review]
Thanks!

**Pull request merged**